portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/structure/WebContentStructures.testcase

```diff
@@ -315,6 +317,26 @@ definition {
 				sitePageType = "Link to URL");
 		}
 
+		task ("Assert cancel edit a Link to URL type page will not redirect to the URL") {
+			SitePages.openPagesAdmin(siteURLKey = "test-site-name");
+
+			SitePages.gotoPageMenuItem(
+				menuItem = "Configure",
+				pageName = "Test Link To URL");
+
+			PortletEntry.inputName(name = "Edit Link to URL Page");
+
+			AssertClick(
+				locator1 = "Button#CANCEL",
+				value1 = "Cancel");
+
+			var portalURL = PropsUtil.get("portal.url");
+			var currentURL = Navigator.getCurrentURL();
+			var p_p_authParam = StringUtil.extractLast("${currentURL}", "=");
+
+			AssertLocation(value1 = "${portalURL}/group/test-site-name/~/control_panel/manage?p_p_id=com_liferay_layout_admin_web_portlet_GroupPagesPortlet&p_p_lifecycle=0&p_p_state=maximized&p_p_auth=${p_p_authParam}");
```

Comment:
@Jayda-4145 this one is also pretty long.  Please use AssertLocation.assertPartialLocation on a small consistent section of it, like the "GroupPagesPortlet" part, then resend when it's working.  Thanks.